hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleHederaStateTest.java

```diff
@@ -449,46 +449,30 @@ void knownServiceNameUsingReadableStates() {
             // Then query it, we find the data we expected to find
             assertThat(states).isNotNull();
             assertThat(states.isEmpty()).isFalse();
-            assertThat(states.size()).isEqualTo(4); // animal and fruit and country
+            assertThat(states.size()).isEqualTo(4); // animal and fruit and country and steam
 
-            final ReadableKVState<String, String> fruitStates = states.get(FRUIT_STATE_KEY);
-            assertThat(fruitStates).isNotNull();
-            assertThat(fruitStates.get(A_KEY)).isSameAs(APPLE);
-            assertThat(fruitStates.get(B_KEY)).isSameAs(BANANA);
-            assertThat(fruitStates.get(C_KEY)).isNull();
-            assertThat(fruitStates.get(D_KEY)).isNull();
-            assertThat(fruitStates.get(E_KEY)).isNull();
-            assertThat(fruitStates.get(F_KEY)).isNull();
-            assertThat(fruitStates.get(G_KEY)).isNull();
-
-            final ReadableKVState<String, String> animalStates = states.get(ANIMAL_STATE_KEY);
-            assertThat(animalStates).isNotNull();
-            assertThat(animalStates.get(A_KEY)).isNull();
-            assertThat(animalStates.get(B_KEY)).isNull();
-            assertThat(animalStates.get(C_KEY)).isSameAs(CUTTLEFISH);
-            assertThat(animalStates.get(D_KEY)).isSameAs(DOG);
-            assertThat(animalStates.get(E_KEY)).isNull();
-            assertThat(animalStates.get(F_KEY)).isSameAs(FOX);
-            assertThat(animalStates.get(G_KEY)).isNull();
+            final ReadableKVState<String, String> fruitState = states.get(FRUIT_STATE_KEY);
```

Comment:
What was Sonar complaining about here? I think the test code is easier to understand the way it was.

Reply:
Test methods should not contain too many assertions: https://sonarsource.atlassian.net/browse/RSPEC-5961